[MLOB] Update Documentation for Configurations Page for ML Observability (#27007)
DataDog/documentation: content/en/llm_observability/configuration/_index.md

For more information about evaluations, see [Terms and Concepts][1].

### Estimated Token Usage

> Datadog provides metrics to help you monitor and manage the token usage associated with evaluations that power LLM Observability (LLM Obs). While LLM Obs enables you to monitor your LLM applications, these metrics allow you to track the resources consumed by the LLMs behind the features of LLM Obs itself, ensuring efficient usage and cost management.

Reviewer: ```suggestion
LLM Observability provides metrics to help you monitor and manage the token usage associated with evaluations that power LLM Observability. The following metrics allow you to track the LLM resources consumed to power evaluations:
```

> The following metrics provide detailed insights into token usage during evaluations:

Reviewer: ```suggestion

```

- `ml_obs.estimated_usage.llm.input.tokens`
- `ml_obs.estimated_usage.llm.output.tokens`
- `ml_obs.estimated_usage.llm.total.tokens`

> Each of these metrics is tagged with `ml_app`, `model_server`, `model_provider`, `model_name`, and `evaluation_name`, allowing you to pinpoint specific applications, models, and evaluations contributing to your usage. These metrics help you maintain visibility over the LLM-powered features in LLM Obs.

Reviewer: ```suggestion
Each of these metrics has `ml_app`, `model_server`, `model_provider`, `model_name`, and `evaluation_name` tags, allowing you to pinpoint specific applications, models, and evaluations contributing to your usage.
```
